### ISSUE TITLE
CLI broken when output path is not set

### ISSUE THREAD

**Issue by zmwangx** [beautify-web/js-beautify#933]
**Update.** Only broken in node 6.0.0 or later.

---

I'm running the latest master (bdb91a2c307cc94c6860cedd4c0149c28762e82d).

On the command line:

    > js-beautify foo.js

Expected: beautified result printed to stdout. Instead:

```
TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.dirname (path.js:1324:5)
    at Object.processInputSync (/usr/local/lib/node_modules/js-beautify/js/lib/cli.js:318:24)
    at Array.forEach (native)
    at Object.exports.interpret (/usr/local/lib/node_modules/js-beautify/js/lib/cli.js:205:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/js-beautify/js/bin/js-beautify.js:4:5)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
Run `js-beautify -h` for help.
```

There's no such error when a custom output path is specified using `-o, --outfile`.

The problem appears to be `config.outfile` being `undefined`.

**Comment by riatzukiza** [comment]
I recieve this same problem. It happens to me when I am using the web-beautify-js tool in spacemacs, which uses js-beautify via CLI to pipe the input into the emacs buffer, this was working until recently, then stopped after I updated node js to 6.0.0. I have updated the application since then and I still recieve this problem.

**Comment by riatzukiza** [comment]
I will back port to an older version of node to see if this is resolved.

**Comment by zmwangx** [comment]
Now that you mention it, yes, it's broken by node 6.0.0. Works fine under 5.11.0.

**Comment by riatzukiza** [comment]
I just downgraded to 4.4.4, the latest stable version of node, and it does indeed work.

**Comment by bitwiseman** [comment]
@riatzukiza, @zmwangx - Thanks for tracking this down to version issues. I'll update the test environments to include nodejs 6.x and see if this can be tracked down.

**Comment by zmwangx** [comment]
Fix in #939.
